DuoUniversal/Client.cs

```diff
@@ -80,14 +80,15 @@ public async Task<bool> DoHealthCheck(bool handleException = true)
         /// </summary>
         /// <param name="username">The username to authenticate.  Must match a Duo username or alias</param>
         /// <param name="state">A unique identifier for the authentication attempt</param>
+        /// <param name="issuer">A specific parameter used for the Epic Hyperdrive integration to generate the samlResponse</param>
         /// <returns>A URL to redirect the user's browser to</returns>
-        public string GenerateAuthUri(string username, string state)
+        public string GenerateAuthUri(string username, string state, string issuer = null)

```

Comment:
Is this issuer value going to be different per request?
Or does it make more sense to make it an optional parameter on the Client object, like the "use duo code attribute"?